### ISSUE TITLE
Make configuration API available post-initialization.

### ISSUE THREAD

**Issue by Steven0351** [SwiftyTesseract/SwiftyTesseract#70]
Since this is now the only way to set tesseract variables, there should also be a member function with a signature that looks similar to this:
```swift
public func configure(@ConfigurationBuilder configuration: () -> (TessBaseAPI) -> Void) {
  configuration()(tesseract)
}
```

**Comment by Steven0351** [comment]
This has been implemented in the Swift 5.3 branch

**Comment by Steven0351** [comment]
In the develop branch from #74
